lib/internal/http2/core.js

```diff
@@ -2348,6 +2367,13 @@ function initializeOptions(options) {
   assertIsObject(options, 'options');
   options = Object.assign({}, options);
   options.allowHalfOpen = true;
+  if (options.slowReadTimeout === undefined)
+    options.slowReadTimeout = kDefaultSlowReadTimeout;
+  if (typeof options.slowReadTimeout !== 'number') {
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE',
+                               'options.slowReadTimeout',
+                               'number');
+  }
```

Comment:
we should validate this a bit more, is `Infinity` allowed? Is a negative number allowed?